selftests/checkall

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 PYTHON=${PYTHON:-python3}
 ERR=()
-RESULTS_DIR=$(./scripts/avocado config | grep datadir.paths.logs_dir | awk '{print $2}')
+RESULTS_DIR=$($PYTHON -m avocado config | grep datadir.paths.logs_dir | awk '{print $2}')
```

Comment:
Ok, but the line getting the python to use feels sub-optimal, specially inside a virtualenv... It should be `PYTHON=${PYTHON:-$(which python)}`. Not related to your change though.